services/libs/aster-std/src/fs/ramfs/fs.rs

```diff
@@ -412,8 +412,8 @@ impl RamInode {
 }
 
 impl PageCacheBackend for RamInode {
-    fn read_page(&self, _idx: usize, _frame: &VmFrame) -> Result<()> {
-        // do nothing
+    fn read_page(&self, _idx: usize, frame: &VmFrame) -> Result<()> {
+        frame.zero();
```

Comment:
Could you add a comment to make it more clear?

```rust
    fn read_page(&self, _idx: usize, frame: &VmFrame) -> Result<()> {
        // Initially, any block/page in a RamFs inode contains all zeros
        frame.zero();
    }
```

----

Here is an issue that is not introduced by this PR. But I am hoping you can address as a separate commit in this PR.

Having a method name starting with a noun looks quite weird. In consistent with Rust's std, such method is called `fill`. For example,  `slice::fill` is defined as the following.

```rust
pub fn fill(&mut self, value: T)
where
    T: Clone,
```

The `fill` method has a more readable name and is more flexible. So I suggest replace `VmFrame::zero` with `VmWriter::fill`.

```rust
/// Fill the remaining space by repeating `value`.
///
/// # Panic
///
/// The size of the remaining space must be a multiple of the size of `value`.
/// Otherwise, the method would panic.
pub fn fill<T: Pod>(&self, value: T) {

}
```

Reply:
Done